packages/main/test/specs/Input.spec.js

```diff
@@ -326,4 +326,15 @@ describe("Input general interaction", () => {
 		assert.notOk(inputPopover.isDisplayedInViewport(), "The inpuit popover is closed as it lost the focus.");
 		assert.ok(helpPopover.isDisplayedInViewport(), "The help popover remains open as the focus is within.");
 	});
-});
\ No newline at end of file
+
+	it("Doesn't remove value on number type input even if you enter a seconda delimiter", () => {
```

Comment:
typo in "second" 
and 
add in addition typing of couple of "dots" (I tried it it works, but to have it covered with this test), because it used to happen with "dots" for some locales.